cogs/bot.py

```diff
@@ -82,6 +82,7 @@ async def info(self, ctx, *, user: Union[discord.Member, FetchUserConverter] = N
         embed.set_author(name=str(user))
 
         embed.add_field(name="ID", value=user.id, inline=False)
+        embed.add_field(name="Avatar", value=f"[Link]({user.avatar_url})", inline=False)
```

Comment:
This does not work on discord.py 2.0. [see docs](https://discordpy.readthedocs.io/en/master/api.html#discord.User.avatar)

Reply:
```suggestion
        embed.add_field(name="Avatar", value=f"[Link]({user.avatar.url})", inline=False)
```